src/config/db.tsx

```diff
@@ -33,11 +33,25 @@ export const documentSchema: RxJsonSchema<DocumentProperties> = {
     },
     document: {
       type: "object"
+    },
+    qrCode: {
+      type: "object",
+      properties: {
+        url: {
+          type: "string"
+        },
+        expiry: {
+          type: "number"
+        }
+      }
     }
   }
 };
 
 export const documentsCollection: RxCollectionCreator = {
   name: "documents",
-  schema: documentSchema
+  schema: documentSchema,
+  migrationStrategies: {
```

Comment:
Shall we skip migration strategies for now as non has been deployed in the wild 😅